poetry/core/factory.py

```diff
@@ -32,7 +34,7 @@ def create_poetry(
         from .poetry import Poetry
         from .pyproject.toml import PyProjectTOML
 
-        poetry_file = self.locate(cwd)
+        poetry_file = self.locate(cwd or Path.cwd())
```

Comment:
perhaps this default should be contained within the `locate` method?

Reply:
Agreed -- let's go ahead and make that change.